L/Libuuid/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "Libuuid"
+version = v"2.34"
+
+# Collection of sources required to build FriBidi
+sources = [
+    "https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v$(version.major).$(version.minor)/util-linux-$(version.major).$(version.minor).tar.xz" =>
+    "743f9d0c7252b6db246b659c1e1ce0bd45d8d4508b4dfa427bbb4a3e9b9f62b5"
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/util-linux-*/
+./configure --prefix=$prefix --host=$target --disable-all-programs --enable-libuuid
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [p for p in supported_platforms() if !(p isa Windows)]
```

Comment:
Here I listed all platforms where this can actually be built.  However we'd need empty tarballs for Windows, too.